### DISCUSSION TITLE
Please help a noob

### DISCUSSION THREAD

**Discussion by Nevinth** [stefansundin/privatkopiera#131] [Q&A]
I finally managed to download and access ffmpeg correctly on my mac. But now when I copy and paste into terminal I get this error message: 

[tcp @ 0x7fbcb2410b40] Connection to tcp://svt-vod-5r.akamaized.net:443 failed: Host is down
https://svt-vod-5r.akamaized.net/d0/se/20220702/14edabf4-bad6-4e4d-977e-bc22a0bc9967/hls-video-avc-1920x1080p25-1394/hls-video-avc-1920x1080p25-1394.m3u8: Host is down

I just really need the Swedish Paw Patrol theme song for my kid :persevere: :see_no_evil:
But whatever episode I choose, I get the "Host is down". What does this mean?

**Comment by stefansundin** [comment]
Hmm, I don't think I've seen that error before. Are you able to watch the episode in a web browser?

I think the error means that the DNS resolution worked but then it isn't responding to your TCP packets. Do you have any kind of firewall or something that might be interfering? Just a guess.

And feel free to post the link to the show and the full command that you're trying to run.

**Reply by Nevinth** [reply]
Yes I can watch it in the browser.
https://www.svtplay.se/video/33458398/paw-patrol/paw-patrol-sasong-1-valparna-raddar-posten?id=8yEomZq


This is the full terminal commands:
Jossans-iMac-Pro-2:~ jossan$ ffmpeg -i "https://svt-vod-5r.akamaized.net/d0/se/20220702/14edabf4-bad6-4e4d-977e-bc22a0bc9967/hls-video-avc-1920x1080p25-1394/hls-video-avc-1920x1080p25-1394.m3u8" -i "https://svt-vod-5r.akamaized.net/d0/se/20220702/14edabf4-bad6-4e4d-977e-bc22a0bc9967/hls-audio-aac-2ch-192/hls-audio-aac-2ch-192.m3u8" -i "https://svt-vod-5r.akamaized.net/d0/se/20220702/14edabf4-bad6-4e4d-977e-bc22a0bc9967/text/text-0.vtt" -vcodec copy -acodec copy "PAW Patrol - Valparna räddar posten.mkv"
ffmpeg version 5.1.1-tessus Copyright (c) 2000-2022 the FFmpeg developers
  built with Apple clang version 11.0.0 (clang-1100.0.33.17)
  configuration: --cc=/usr/bin/clang --prefix=/opt/ffmpeg --extra-version=tessus --enable-avisynth --enable-fontconfig --enable-gpl --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libfreetype --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenh264 --enable-libopenjpeg --enable-libopus --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvmaf --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-version3 --pkg-config-flags=--static --disable-ffplay
  libavutil      57. 28.100 / 57. 28.100
  libavcodec     59. 37.100 / 59. 37.100
  libavformat    59. 27.100 / 59. 27.100
  libavdevice    59.  7.100 / 59.  7.100
  libavfilter     8. 44.100 /  8. 44.100
  libswscale      6.  7.100 /  6.  7.100
  libswresample   4.  7.100 /  4.  7.100
  libpostproc    56.  6.100 / 56.  6.100
[tcp @ 0x7ff9e25087c0] Connection to tcp://svt-vod-5r.akamaized.net:443 failed: Host is down
https://svt-vod-5r.akamaized.net/d0/se/20220702/14edabf4-bad6-4e4d-977e-bc22a0bc9967/hls-video-avc-1920x1080p25-1394/hls-video-avc-1920x1080p25-1394.m3u8: Host is down

**Reply by Nevinth** [reply]
Oh and I don't think I have a firewall... Not that I installed myself any way

**Comment by Nevinth** [comment]
Anyone who has any idea? Or maybe if someone could try my link and see if it works for them?
